sumpy/symbolic.py

```diff
@@ -243,6 +244,30 @@ def find_power_of(base, prod):
     return result[power]
 
 
+class SpatialConstant(prim.Variable):
```

Comment:
Hook this into the documentation and write a docstring explaining the motivation.

Reply:
Done in 8e7435a. Ready for review